Products/membrane/at/interfaces.py

```diff
@@ -41,7 +41,20 @@ def verifyCredentials(credentials):
         """
 
 
-class IPropertiesProvider(IReferenceable, IBaseObject):
+class IUserPropertiesProvider(IReferenceable, IBaseObject):
+    """
+    Marks the object as a Membrane properties provider using the
+    default properties computation mechanism defined in the Properties
+    adapter (i.e. 'user_property' attribute on the schema fields).
+    """
+
+
+# Until Products.membrane 2.x we only had IPropertiesProvider.
```

Comment:
Actually, we can indeed make this comment clearer.  Done and pushed to master. It is now: 'Before Products.membrane 3.x we only had IPropertiesProvider.'